src/swagger.yml

```diff
@@ -2,29 +2,46 @@ swagger: '2.0'
 info:
   description: RESTful API for the Denner Portal
   version: 1.0.0
-  title: Denner Portal 2.0
+  # Caution: For development imports in API Gateway use your personal name acronym as version, i.e. "denner-api-iw"
+  title: "denner-api-cd"
```

Comment:
Before I'll merge this PR, this will be set to "denner" to reflect the production name in API-Gateway (i do not change it now to be sure I do not overwrite the production api 😁)